docs/docs/development/hardware-integration/dongle.md

```diff
@@ -0,0 +1,203 @@
+        kscan = <&kscan0>;
+        keys = <
+            // Really long list of keys. Keys property may not be present.
+        >;
+    };
+```
+
+You may find some keyboards import their physical layouts from [this location](https://github.com/zmkfirmware/zmk/tree/main/app/dts/layouts).
+
+Now that you've found these nodes, make a file called `my_keyboard_dongle.overlay` (renamed as appropriate) and add the following lines to it:
+
+```dts title="my_keyboard_dongle.overlay"
+/ {
+    chosen {
+        zmk,kscan = &mock_kscan;
+        zmk,physical-layout = &default_layout;
+    };
+
+    mock_kscan: mock_kscan_0 {
+        compatible = "zmk,kscan-mock";
+        columns = <0>;
+        rows = <0>;
+        events = <0>;
+    };
+
+    // Copy of matrix transform node here
+    // Copy of physical layout node here
+      // If kscan property of physical layout is present, assign &mock_kscan to it
+};
+```
+
+Make sure that the matrix transform is selected by the physical layout. If the physical layout imported, you may need to override it:
+
+```dts title="my_keyboard_dongle.overlay"
+&default_layout {
+    transform = <&default_transform>;
+};
+```
+
+If no physical layout is found, you should skip it and instead select the matrix transform as chosen directly:
+
+```dts title="my_keyboard_dongle.overlay"
+/ {
+    chosen {
+        zmk,kscan = &mock_kscan;
+        zmk,matrix-transform = &default_transform;
+    };
+};
+```
+
+:::note
+If multiple physical layouts and matrix transformations are present, you should only need to import the one you wish to use. However, if you are getting errors, then the shield might be defining things in an unusual manner. In this case, copy in all matrix transformations and physical layouts, overriding as appropriate.
+:::
+
+## Enabling the Dongle
+
+Next, navigate to the `config` folder found in your `zmk-config`'s root.
+
+If your keyboard is a unibody, add the following to your `my_keyboard.conf` file (create the file if it does not exist):
+
+```kconfig title="config/my_keyboard.conf"
+# Enable split functionality, this will put the unibody keyboard in peripheral mode
+CONFIG_ZMK_SPLIT=y
+```
+
+If your keyboard is a split keyboard, add the following to your `my_keyboard_left.conf` file, assuming `_left` is the current central (create the file if it does not exist):
+
+```kconfig title="config/my_keyboard_left.conf"
+# Disable central role to use dongle
+CONFIG_ZMK_SPLIT_ROLE_CENTRAL=n
+```
+
+If your keyboard is a split keyboard, you will also need to increase the number of peripherals the dongle can connect to.
+Make a file called `my_keyboard_dongle.conf` and add the following lines to it:
+
+```kconfig title="config/my_keyboard_dongle.conf"
+CONFIG_ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS=n # Change n to the number of peripherals your dongle will have
+CONFIG_BT_MAX_CONN=m # Change m to be equal or greater than the number of peripherals + the number of bt hosts the dongle should connect to
```

Comment:
IIRC you can't have comments on the same line following the Kconfig value, so we'd need to move them to separate lines.